api/FileReaderSync.json

```
@@ -249,103 +249,59 @@
             },
             "chrome_android": {
               "version_added": true,
-              "version_removed": "59"
+              "notes": "Not supported in service workers from Chrome 59"
             },
             "edge": {
-              "version_added": null
+              "version_added": "12",
+              "notes": "Not supported in service workers"
             },
             "firefox": {
-              "version_added": true,
-              "version_removed": "61"
+              "version_added": "8",
+              "notes": "Not supported in service workers from FF61"
             },
             "firefox_android": {
-              "version_added": true,
-              "version_removed": "61"
+              "version_added": "8",
+              "notes": "Not supported in service workers from FF61"
             },
             "ie": {
-              "version_added": false
+              "version_added": true,
+              "notes": "Not supported in service workers"
             },
             "opera": {
               "version_added": true,
-              "version_removed": "46"
+              "notes": "Not supported in service workers from Opera 46"
             },
             "opera_android": {
               "version_added": true,
-              "version_removed": "43"
+              "notes": "Not supported in service workers from Opera 43"
             },
             "safari": {
-              "version_added": false
-            },
-            "safari_ios": {
-              "version_added": false
-            },
-            "samsunginternet_android": {
               "version_added": true,
-              "version_removed": "7.0"
-            },
-            "webview_android": {
-              "version_added": true,
-              "version_removed": "59"
-            }
-          },
-          "status": {
-            "experimental": false,
-            "standard_track": false,
-            "deprecated": true
-          }
-        }
-      },
-      "worker_support": {
-        "__compat": {
-          "description": "Available in workers",
-          "support": {
-            "chrome": {
-              "version_added": true
-            },
-            "chrome_android": {
-              "version_added": true
-            },
-            "edge": {
-              "version_added": "12"
-            },
-            "firefox": {
-              "version_added": "8"
-            },
-            "firefox_android": {
-              "version_added": "8"
-            },
-            "ie": {
-              "version_added": true
-            },
-            "opera": {
-              "version_added": true
-            },
-            "opera_android": {
-              "version_added": true
-            },
-            "safari": {
-              "version_added": true,
-              "version_removed": "6.1"
+              "version_removed": "6.1",
+              "notes": "Service workers not supported"
```

suggestion: for consistency with other notes in this PR

```suggestion
              "notes": "Not supported in service workers."
```